tests/e2e/core-tests/specs/shopper/front-end-checkout-registered.test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, jest/expect-expect, jest/no-standalone-expect */
+/**
+ * Internal dependencies
+ */
+const {
+	shopper,
+	merchant,
+	createSimpleProduct,
+	setCheckbox,
+	settingsPageSaveChanges,
+	uiUnblocked,
+	verifyCheckboxIsSet
+} = require( '@woocommerce/e2e-utils' );
+
+const config = require( 'config' );
+const simpleProductName = config.get( 'products.simple.name' );
+const singleProductPrice = config.has('products.simple.price') ? config.get('products.simple.price') : '9.99';
+
+let orderId;
+
+const runCheckoutRegisteredPageTest = () => {
+	describe('Checkout page', () => {
+		beforeAll(async () => {
+			await merchant.login();
```

Comment:
The existing shopper tests depend on these settings already being set. If we were to update all the shopper tests to include enabling these settings using the browser it would add a significant amount of runtime. I think we should defer making the shopper test atomic until we have a function to do it with the api package.